src/System.Threading.Tasks.Dataflow/src/System.Threading.Tasks.Dataflow.csproj

```diff
@@ -49,6 +50,8 @@
     <Compile Include="Internal\SpscTargetCore.cs" />
     <Compile Include="Internal\TargetCore.cs" />
     <Compile Include="Internal\TargetRegistry.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard1.0' OR '$(TargetGroup)' == 'netstandard1.1'">
```

Comment:
Because the condition here is negation of the condition above. I basically just copied the pattern used for `ConcurrentQueue`:

* In `DefineConstants`, have `Condition` that defines the constant for new configurations.
* In `ItemGroup` have `Condition` that includes the files for old configurations.